webpack.config.js

```diff
@@ -1,15 +1,28 @@
-const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const MiniCssExtractPlugin = require( 'mini-css-extract-plugin' );
 const defaultConfig = require( '@wordpress/scripts/config/webpack.config' );
 
+// update resource asset to have public path inside plugin
+// required for correctly serving assets in production
+defaultConfig.module.rules
+	.filter( ( r ) => r.type === 'asset/resource' )
+	.forEach( ( r ) => {
+		r.generator.publicPath = '/wp-content/plugins/blocks-everywhere/build/';
```

Comment:
Can you expand more about what this is doing? I'm not sure we can guarantee that this path will always be valid.